### PR TITLE
Tailwind CSS installed in Rails project.

### PR DESCRIPTION
**Pull request by ErikGmz**
Tailwind CSS was installed in Rails project and tested through placeholder TestsController and the correspoding view (these resources will be subsequently deleted).

image

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -6,5 +6,5 @@
   get "up" => "rails/health#show", as: :rails_health_check
```
Comment (MauricioNC) [inline config/routes.rb]: Delete unnecessary routes

**Review by MauricioNC — CHANGES_REQUESTED** [review]

File: config/routes.rb (hunk 1 above)
Comment (ErikGmz) [reply config/routes.rb]: Done.

**Review by MauricioNC — APPROVED** [review]
LGTM!

File: test/controllers/tests_controller_test.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class TestsControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
```
Comment (juperameza) [inline test/controllers/tests_controller_test.rb]: If youre not going to use this delete it

**Review by juperameza — CHANGES_REQUESTED** [review]

File: test/controllers/tests_controller_test.rb (hunk 2 above)
Comment (ErikGmz) [reply test/controllers/tests_controller_test.rb]: Fixed!

**Review by juperameza — APPROVED** [review]
LGTM

**Pull request merged**